executors/multiversx/module/scCallsModule_test.go

```diff
@@ -59,73 +57,59 @@ func TestNewScCallsModule(t *testing.T) {
 	t.Run("nil proxy should error", func(t *testing.T) {
 		t.Parallel()
 		cfg := createTestConfigs()
-		module, err := NewScCallsModule(cfg, nil, &testsCommon.LoggerStub{})
+		argsScCallsModule := ArgsScCallsModule{
+			Cfg:   cfg,
+			Proxy: nil,
+			Log:   &testsCommon.LoggerStub{},
+		}
+
+		module, err := NewScCallsModule(argsScCallsModule)
 		assert.NotNil(t, err)
-		assert.Contains(t, err.Error(), "nil proxy")
+		assert.Equal(t, err, errNilProxy)
```

Comment:
replace err with errNilProxy? Reason: the second parameter is the expected one, the last is the actual one.

Also on L82